Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/DocumentPropertyEditor/PrefabOverrideLabelHandler.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzToolsFramework/Prefab/DocumentPropertyEditor/PrefabOverrideLabelHandler.h>
+
+namespace AzToolsFramework::Prefab
+{
+    PrefabOverrideLabelHandler::PrefabOverrideLabelHandler()
+        : m_overridden(false)
+    {
+        m_iconButton = new QToolButton();
```

Comment:
nit: we usually do these in the initializer list above, like:
, m_iconButton(new QToolButton())
, m_textLabel(new AzQtComponents::ElidingLabel())